api/src/test/java/org/pmiops/workbench/notebooks/NotebooksServiceTest.java

```diff
@@ -116,14 +150,36 @@ public void testGetReadOnlyHtml_allowsDataImage() {
 
   @Test
   public void testGetReadOnlyHtml_disallowsRemoteImage() {
-    when(firecloudService.getWorkspace(any(), any()))
+    when(mockFirecloudService.getWorkspace(any(), any()))
         .thenReturn(
             new FirecloudWorkspaceResponse().workspace(new FirecloudWorkspace().bucketName("bkt")));
-    when(cloudStorageService.getFileAsJson(any(), any())).thenReturn(new JSONObject());
-    when(firecloudService.staticNotebooksConvert(any()))
+    when(mockCloudStorageService.getFileAsJson(any(), any())).thenReturn(new JSONObject());
+    when(mockFirecloudService.staticNotebooksConvert(any()))
         .thenReturn("<img src=\"https://eviltrackingpixel.com\" />\n");
 
     String html = new String(notebooksService.getReadOnlyHtml("", "", "").getBytes());
     assertThat(html).doesNotContain("eviltrackingpixel.com");
   }
+
+  @Test
```

Comment:
love how clean these are

Reply:
Thanks. I'm a complete slob in every area except code.